LICENSE.txt

```diff
@@ -2,6 +2,7 @@
 This project is available under the following license:
 ************************************
 Copyright 2012-2016 eBusiness Information
+Copyright 2016-2017 the AndroidAnnotations project
```

Comment:
Not sure if you are mistaking this file with `AndroidAnnotations\src\etc\header.txt` ?